services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -61,16 +63,12 @@ object ProjectFileService {
       projectFile: ProjectFile,
       version: Option[PipelineVersion],
       userId: UserId
-    ): Future[Either[VersioningException, UpdateFiledResponse]] =
+    ): Future[Either[VersioningException, Unit]] =
```

Comment:
uploadFile in ProjectFileController still has `uploadResponse` as variable name, if that is even an issue

Reply:
Great catch, thanks! Although I'm disapppointed that our tests didn't find it. Anyway, I've added third commit that resolves it.